setup.py

```diff
@@ -456,7 +475,7 @@ def make_rpm():
     SOURCES = RPMBUILD+"/SOURCES"
     if not os.path.exists(SOURCES):
         os.makedirs(SOURCES, 0o755)
-    os.rename("dist/" + tarxz, SOURCES + tarxz)
+    os.rename("dist/" + tarxz, os.path.join(SOURCES, tarxz))
```

Comment:
This change is unrelated,  I've rolled it into: 0c3ba3b777673846ad377211975c9bd50f1a3ea3

Reply:
Done!